src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -69,12 +77,19 @@ const styles = ({ spacing, palette }: Theme) =>
       }
     },
     outlinedInput: {
-      paddingLeft: 12
+      '& > div:last-child > div': {
+        padding: 0
+      }
+    },
+    filledInput: {
+      '& > div:first-child > div': {
+        transform: 'none'
+      }
     }
   });
 
 function InputComponent({ inputRef, ...rest }: any) {
-  return <div style={{ height: 'auto' }} ref={inputRef} {...rest} />;
```

Comment:
Hope there is no issue removing this, I remember having to add it for...some reason. I don't remember why sadly.

Reply:
When I removed it nothing happened, but I thought the same thing. I can't remember why it was added. Might have been something with a previous version of material-ui.